docs/modules/ROOT/pages/objc.adoc

```diff
@@ -790,6 +790,24 @@ In the `OrderBy` array, use a string of the form `Rank(X)`, where `X` is
 the property or expression being searched, to represent the ranking of
 the result.
 
+Live Query
+~~~~~~~~~~
+
+A live query stays active and monitors the database and query index for changes. When there's a change it re-runs itself automatically, and if the query results changed it notifies any observers. A live query is a great way to build reactive user interfaces, especially table/list views, that keep themselves up to date. For example, as the replicator runs and pulls new data from the server, a live query-driven UI will automatically update to show the data without the user having to manually refresh. This helps your app feel quick and responsive.
```

Comment:
To be precise, Live Query doesn't modify query index, it just monitors database changes.